### PR TITLE
Add a console design.

### PR DESCRIPTION
**Pull request by Artemis21**
Like the Windows 95 theme but unlike Galaxy, Hotdog or Unicorn, this theme is meant to look like something old rather than look like an old design. It is green-on-dark though, and that is perhaps too similar to Galaxy?

### CODE REVIEW

File: jolly-jellyfish/src/static/themes/console.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  padding-left: 1ch;
+}
+  
+@keyframes typing {
+  0% {max-width: 0}
+  50% {max-width: 100ch}
+  100% {max-width: 0}
+}
+
+h1::before, h2::before, h3::before, h4::before, h5::before {
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/console.css]: you're missing `h6`

Comment (A5rocks) [reply jolly-jellyfish/src/static/themes/console.css]: Also, you could make this bold maybe? I'm not sure.

File: jolly-jellyfish/src/static/themes/console.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+p::before {
+  content: "> ";
+}
+
+body > *:nth-last-child(1)::after {
+  content: " █";
+  animation: blink 1s infinite steps(1);
+}
+
+h1::after, h2::after, h3::after, h4::after, h5::after {
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/console.css]: same here, you're missing h6

File: jolly-jellyfish/src/static/themes/console.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  background: black;
+  color: #0f0;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  display: flex;
+}
+
+h1 > span > span, h2 > span > span, h3 > span > span, h4 > span > span, h5 > span > span, h6 > span > span {
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/console.css]: doubly nested spans? why?

File: jolly-jellyfish/src/static/themes/console.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
+/**
+ * Green on black monospace theme.
+ * Every paragraph begins with "> " and every header with ">>> ".
+ * The final paragraph ends with a blinking cursor, as well as every header.
+ * Headers are constantly deleted and rewritten, and must be written as:
+ *     <h1><span><span>Header Text</span</span></h1>
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/console.css]: Are you sure we can't remove one of these spans? If we're using them as unique ids then we could drop a class on the outer one and use `h1 > span.class` (I think... CSS is not my forte)

**Review by A5rocks — COMMENTED** [review]
It would be nice to have this up in some site so we can see how it looks and offer feedback without requiring us to `python -m http.server 8080` in this directory and write the html ourselves. :P

File: jolly-jellyfish/src/static/themes/console.css (hunk 4 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Only one is needed here, but two are required for the hotdog theme.

File: jolly-jellyfish/src/static/themes/console.css (hunk 1 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Yeah I'm missing h6 in quite a few actually, I forgot about that. Most browsers make headers bold by default AFAIK.

**Comment by Artemis21** [comment]
> It would be nice to have this up in some site so we can see how it looks and offer feedback without requiring us to python -m http.server 8080 in this directory and write the html ourselves. :P

That would be nice, but you don't have to make a server anyway - you should just be able to open the local file in your browser.

File: jolly-jellyfish/src/static/themes/console.css (hunk 3 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Resolved as raised twice

File: jolly-jellyfish/src/static/themes/console.css (hunk 2 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Resolved as raised twice

File: jolly-jellyfish/src/static/themes/console.css (hunk 4 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: For simplicity, I will only select the first one here though, that is a good idea

File: jolly-jellyfish/src/static/themes/console.css (hunk 1 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Added h6

File: jolly-jellyfish/src/static/themes/console.css (hunk 4 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css]: Done

**Pull request merged**